benchmarks/dynamo/common.py

```diff
@@ -93,6 +93,22 @@
     # ignore the error if torch_xla is not installed
     pass
 
+def is_hpu():
```

Comment:
Why do we test hpu from sys.argv? Can we test device as an arg instead?

Reply:
In our case device arg is cuda, we need to work around that.

Reply:
If you parse from the argparser, it would be hpu or cuda? I'm wondering why not reuse argparser here.

Reply:
Yes, a separate instance of argparse used only for this function should work just fine.

Reply:
Done.